code/src/main/java/com/google/sps/storage/messaging/ContactStatus.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.storage;
+
+import java.util.Optional;
+
+/**
+* Interface to help identify if a notification was successful.
+*/
+public interface ContactStatus {
+  public void setSuccess();
```

Comment:
each of the names in this interface could be a bit better. What about:

markContactedSuccessfully()
hasBeenContactedSuccessfully()
getTimeWhenContacted()

Reply:
Why change getTimeWhenContactedSeconds to getTimeWhenContacted, isnt that making it less descriptive?